src/index.tsx

```diff
@@ -26,10 +26,10 @@ import "./styles/viz-workaround.css"
 
 import { store } from "./store"
 
-import { setInitialSiteCookie } from "./utils/switch-site"
+import { syncInitialSiteCookie } from "./utils/switch-site"
 import { ThemeProvider } from "./components/ThemeProvider"
 
-setInitialSiteCookie()
+syncInitialSiteCookie()
```

Comment:
This is from https://github.com/metabase/shoppy/pull/69

Reply:
Is it needed on this pr?

Reply:
@npretto removed from this PR in https://github.com/metabase/shoppy/pull/71/commits/4e688df9448b307b4b8c1f3a495026e3d9bf4ed6 - as we use query parameters for cacheability instead of cookies, I got rid of cookies entirely.